File: C/Arreglos/ARR_DEF_CONCEPT_1.c
#include <stdio.h>
#include <stdlib.h> 

void CARGA_TECLADO_VECTOR(int v[5]) {
	int i;

	for(i = 0; i < 5; ++i) {
		printf("Ingrese valor para posicion %d :", i+1);
		scanf("%d", &v[i]);		
		}
}

void CARGA_TECLADO_MATRIZ(int m[3][3]) {
	int i, j;

	for(i = 0; i < 3; ++i) {
		for(j = 0; j < 3; ++j) {
			printf("Ingrese valor para fila %d -- col %d :", i+1, j+1);
			scanf("%d", &m[i][j]);			
			}
		}
}

int main() {	
	int i, j;
	int V[5];
	int M[3][3];

	CARGA_TECLADO_VECTOR(V);

	CARGA_TECLADO_MATRIZ(M);

	printf(" ---------------------\n");
	printf(" ----- RESULTADOS ----\n");
	printf(" ---------------------\n");

	printf("Vector: \n");

	//valores de los elementos del vector
	for(i = 0; i < 5; ++i) {
		printf("%d ", V[i]);
	}

	printf("\n---------------------\n");
	printf("Matriz: \n");

	for(i = 0; i < 3; ++i) {
		for(j = 0; j < 3; ++j) {
			printf("%d ", M[i][j]);
		}
		printf("\n");
	}

	return 0;
}

File: Python/Arreglos/ARR_DEF_CONCEPT_1.py
def CARGA_TECLADO_VECTOR(v):
    for i in range(len(v)):
        v[i] = int(input(f"Ingrese valor para posicion {i+1}: "))

def CARGA_TECLADO_MATRIZ(m):
    for i in range(len(m)):
        for j in range(len(m[i])):
            m[i][j] = int(input(f"Ingrese valor para fila {i+1} -- col {j+1}: "))

def main():
    V = [0]*5
    M = [[0]*3 for _ in range(3)]
    CARGA_TECLADO_VECTOR(V)
    CARGA_TECLADO_MATRIZ(M)
    print(" ---------------------")
    print(" ----- RESULTADOS ----")
    print(" ---------------------")
    print("Vector: ")
    for val in V:
        print(val, end=' ')
    print()
    print("---------------------")
    print("Matriz: ")
    for fila in M:
        for val in fila:
            print(val, end=' ')
        print()
if __name__ == "__main__":
    main()
